### PR TITLE
Feature/prettier eslint

### PR DESCRIPTION
**Pull request by pablo-rodriguez-jd**

### CODE REVIEW

File: generators/app/templates/src/components/NavBar/children/Toolbar/toolbar.test.js [hunk 1]
```diff
@@ -1,22 +1,21 @@
-import React from "react";
+/* eslint-disable jsx-a11y/label-has-associated-control */
```
Comment (james-martin-jd) [inline generators/app/templates/src/components/NavBar/children/Toolbar/toolbar.test.js]: What is the reason for disabling these in this file?

File: generators/app/templates/src/components/Utils/Loader/loader.component.js [hunk 2]
```diff
@@ -24,6 +27,7 @@ class Loader extends Component<Props> {
         <LoaderWrapper>
           <CubeGrid>
             {cubes.map((delay, i) => (
+              // eslint-disable-next-line react/no-array-index-key
```
Comment (james-martin-jd) [inline generators/app/templates/src/components/Utils/Loader/loader.component.js]: Is disable required? Is there a way to work around this?

File: generators/app/templates/src/containers/Register/register.component.js [hunk 3]
```diff
@@ -100,15 +89,16 @@ class RegisterComponent extends Component<Props, State> {
...
                     </a>
                     <Link to="/login" className="a-with-spacing">
-                      {t("register.alreadySolid")}
+                      {t('register.alreadySolid')}
                     </Link>
                     <ul>
                       {providers.map((providerData, i) => (
                         <ProviderItem
                           data={providerData}
+                          // eslint-disable-next-line react/no-array-index-key
```
Comment (james-martin-jd) [inline generators/app/templates/src/containers/Register/register.component.js]: Is there a way to remove the need for this disable?

File: generators/app/templates/src/serviceWorker.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (james-martin-jd) [inline generators/app/templates/src/serviceWorker.js]: Is there a way to remove the need for this disable?

File: generators/app/templates/test/__mocks__/solid-auth-client.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable class-methods-use-this */
```
Comment (james-martin-jd) [inline generators/app/templates/test/__mocks__/solid-auth-client.js]: Is there a way to remove the need for this disable?

File: generators/app/templates/scripts/build.js [hunk 6]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```
[the remark this answers is not in the document]
Comment (pablo-rodriguez-jd) [reply generators/app/templates/scripts/build.js]: This file is specific to CRA, therefore we probably should not touch it

File: generators/app/templates/src/containers/Register/register.component.js (hunk 3 above)
Comment (pablo-rodriguez-jd) [reply generators/app/templates/src/containers/Register/register.component.js]: Fixed

File: generators/app/templates/src/components/Utils/Loader/loader.component.js (hunk 2 above)
Comment (pablo-rodriguez-jd) [reply generators/app/templates/src/components/Utils/Loader/loader.component.js]: Fixed

File: generators/app/templates/src/components/NavBar/children/Toolbar/toolbar.test.js (hunk 1 above)
Comment (pablo-rodriguez-jd) [reply generators/app/templates/src/components/NavBar/children/Toolbar/toolbar.test.js]: Fixed

File: generators/app/templates/src/components/AuthNavBar/auth-nav-bar.component.js [hunk 7]
```diff
@@ -1,37 +1,44 @@
-import React from "react";
-import { NavBar } from "@components";
+/* eslint-disable no-unused-expressions */
```
[the remark this answers is not in the document]
Comment (pablo-rodriguez-jd) [reply generators/app/templates/src/components/AuthNavBar/auth-nav-bar.component.js]: Removed

File: generators/app/templates/src/serviceWorker.js (hunk 4 above)
Comment (pablo-rodriguez-jd) [reply generators/app/templates/src/serviceWorker.js]: This file is specific to CRA, therefore we probably should not touch it

File: generators/app/templates/test/__mocks__/solid-auth-client.js (hunk 5 above)
Comment (pablo-rodriguez-jd) [reply generators/app/templates/test/__mocks__/solid-auth-client.js]: Removed

**Pull request merged**